target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -12,6 +12,13 @@ asus,rt-n56u-b1)
 	ucidef_set_led_netdev "lan" "LAN link" "blue:lan" "br-lan"
 	ucidef_set_led_netdev "wan" "WAN link" "blue:wan" "wan"
 	;;
+cudy,wr2100)
+	ucidef_set_led_netdev "lan1" "lan1 link" "green:lan1" "lan1"
+	ucidef_set_led_netdev "lan2" "lan2 link" "green:lan2" "lan2"
+	ucidef_set_led_netdev "lan3" "lan3 link" "green:lan3" "lan3"
+	ucidef_set_led_netdev "lan4" "lan4 link" "green:lan4" "lan4"
+	ucidef_set_led_netdev "wan" "wan link" "green:wan" "wan"
```

Comment:
Adding link to the name seems strange here, as this will also show activity?

I'd just call it lan1, lan2, etc. ?

Reply:
oops! yeah.
sloppy copy mistake.